libcodechecker/analyze/analyzers/result_handler_base.py

```diff
@@ -176,10 +176,10 @@ def analyzer_result_file(self):
             analyzed_file = self.analyzed_source_file
             _, analyzed_file_name = os.path.split(analyzed_file)
 
-            uid = str(uuid.uuid1()).split('-')[0]
+            file_name = str(self.buildaction.analyzer_type) + '_' + \
+                self.__buildaction.original_command
 
-            out_file_name = str(self.buildaction.analyzer_type) + \
-                '_' + analyzed_file_name + '_' + uid + '.plist'
+            out_file_name = hashlib.md5(file_name).hexdigest() + '.plist'
```

Comment:
Besides the hash, keep the source file name also in the plist file name to help the users to identify which plist file belongs to which source file. 
So use something like: out_file_name = analyzed_file_name + hashlib.md5(file_name).hexdigest() + '.plist'